### DISCUSSION TITLE
Error after upgrading to 0.22

### DISCUSSION THREAD

**Discussion by vicdavery** [macxq/foxess-ha#140] [General]
Hi,
I've just upgraded from 0.15 to 0.22. I've updated my configuration.yaml with the following:

foxess:
  MYFOXLOGIN:
   password: mypassword
   devices:
      FoxESS: 9548f50d-33dd-4697-*****-**********

I get the following error after restart:

foxess: No setup or config entry setup function defined.

Can anyone help? 
Thanks.

**Comment by macxq** [comment]
keep the old configuration format

**Reply by vicdavery** [reply]
Thanks for the quick reply. 
I've reverted to the old config and now get when trying to restart:
[140429346931808] The system cannot restart because the configuration is not valid: Platform error sensor.foxess - cannot import name 'UnitOfEnergy' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
16:04:23 – (ERROR) Home Assistant WebSocket API

**Reply by vicdavery** [reply]
My fault I was using an old version of HA.
